src/main/java/team18/team18_be/apply/controller/ApplyController.java

```diff
@@ -24,17 +27,32 @@ public ApplyController(ApplyService applyService) {
     this.applyService = applyService;
   }
 
-  @PostMapping
+  @PostMapping("/{recruitmentId}")
   public ResponseEntity<Void> createApplicationForm(
-      @RequestBody ApplicationFormRequest applicationFormRequest, @LoginUser User user) {
-    Long applicationId = applyService.createApplicationForm(applicationFormRequest);
+      @RequestBody ApplicationFormRequest applicationFormRequest, @PathVariable Long recruitmentId,
+      @LoginUser User user) {
+    Long applicationId = applyService.createApplicationForm(applicationFormRequest, recruitmentId,
+        user);
     URI location = URI.create("/api/application/" + applicationId);
 
     return ResponseEntity.created(location).build();
   }
 
-  @GetMapping
-  public ResponseEntity<List<ApplyResponse>> SearchApplicant(@LoginUser User user) {
-    return ResponseEntity.ok(applyService.searchApplicacnt());
+  @GetMapping("/{recruitmentId}")
+  public ResponseEntity<List<ApplierPerRecruitmentResponse>> searchApplicant(
+      @PathVariable Long recruitmentId,
+      @LoginUser User user) {
+    return ResponseEntity.ok(applyService.searchApplicant(recruitmentId, user));
+  }
```

Comment:
이런식으로 코드를 수정하여 가독성과 타입을 한번에 확인할 수 있도록 하는게 어떨까요?
```suggestion
  public ResponseEntity<List<ApplierPerRecruitmentResponse>> searchApplicant(
      @PathVariable Long recruitmentId,
      @LoginUser User user) {
    List<ApplierPerRecruitmentResponse> applicants = applyService.searchApplicant(recruitmentId, user);
    return ResponseEntity.ok(applicants);
```

Reply:
네! 반영하겠습니다